scripts/publish-documentation/README.md

```diff
@@ -22,9 +34,9 @@ Prerequisites for the target repository:
 - have at least one tag and module.
 
 Prerequisites for running the script:
-- have [`git`](https://git-scm.com/downloads) and [`jenv`](https://github.com/jenv/jenv#12-adding-your-java-environment) installed;
-- have Java **1.8** installed and [registered](https://github.com/jenv/jenv#12-adding-your-java-environment) in `jenv`;
-- have Java **11** installed, registered and [set as global](https://github.com/jenv/jenv#13-setting-a-global-java-version) in `jenv`.
+- have [`git`](https://git-scm.com/downloads) and [`jenv`][jenv-add] installed;
```

Comment:
Initially, on line 37, I wanted to have the link to the repository without highlighting any specific part of the documentation. And on line 38, the link to the specific section. That was my intention for having two separate links.

Reply:
I'll add the repo link to come before the "add" thing.